resources/lib/service.py

```diff
@@ -20,6 +21,22 @@ def __init__(self):
             if self.waitForAbort(10):
                 break
 
+    @staticmethod
+    def onNotification(sender, method, data):  # pylint: disable=invalid-name
+        ''' Handler for notifications '''
+        _kodi = KodiWrapper(None)
+        _kodi.log('Got a notification: %s, %s, %s' % (sender, method, to_unicode(data)), 'Verbose')
```

Comment:
I would prefer this gets removed eventually, as it harms troubleshooting/testing on Kodi (i.e. unrelated notifications).

Reply:
Okay, when I'll remove the draft status, this will be gone.